pkgs/development/python-modules/keyring-pass/default.nix

```diff
@@ -0,0 +1,83 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, gnupg
+, keyring
+, pass
+, poetry-core
+, pythonOlder
+}:
+let
+  pname = "keyring-pass";
+  version = "0.8.1";
+  module_name = builtins.replaceStrings [ "-" ] [ "_" ] pname;
+in
+
+buildPythonPackage {
+  inherit pname version;
+  disabled = pythonOlder "3.6";
+  format = "pyproject";
+
+  src = fetchPypi {
```

Comment:
I prefer `fetchFromGitHub` but it's up to you.

Reply:
I don't actually know how to build poetry package without `poetry2nix`, but otherwise wouldn't be opposed to `fetchFromGitHub`

Reply:
Poetry-core should be enough. You can simply replace the `src`.